CustomShipSelect.cpp

```diff
@@ -909,6 +909,15 @@ void CustomShipSelect::OnInit(ShipSelect* shipSelect_)
                 {
                     aButton->bShipLocked = !CustomShipUnlocks::instance->GetCustomShipUnlocked(shipName);
                     aButton->bLayoutLocked = aButton->bShipLocked;
+
+                    if (aButton->bShipLocked){
```

Comment:
Brackets should respect the formatting of existing code. Change from
```c++
if (aButton->bShipLocked){
```
to
```c++
if (aButton->bShipLocked)
{
```
for all instances.